readme.MD

```diff
@@ -22,3 +26,333 @@ $ docker-compose up
 ```
 
 NOTE! When installing new dependencies, use the `--build` flag at the end of the command to update the image.
+
+# Demo
+
+[Try the project on Heroku!](https://bulletin-board-onboarding-api.herokuapp.com/)
+
+## /users/
+
+#### GET
+
+```json
+[
+  { "avatar": "Some link to image", "id": 1 },
+  { "avatar": "Test data", "id": 3 },
+  { "avatar": "Test data", "id": 4 },
+  { "avatar": "avatar", "id": 5 }
+]
+```
+
+#### POST
+
+Body
+
+```json
+{
+  "avatar": "New avatar"
+}
+```
+
+Responce
+
+```json
+{
+  "avatar": "New avatar",
+  "id": 6
+}
+```
+
+## /users/:id
+
+Example: /users/1
+
+#### GET
+
+```json
+{
+  "avatar": "Some link to image",
+  "id": 1
+}
+```
+
+#### PUT
+
+Body
+
+```json
+{
+  "avatar": "Different avatar"
+}
+```
+
+Responce
+
+```json
+{
+  "avatar": "Different avatar",
+  "id": 1
+}
+```
+
+#### DELETE
+
+`Status: 204 No Content`
+
+## /posts/
```

Comment:
Can you move Post and Comment endpoint before Users endpoints because we dont have real users yet so they are more important :)